pkg/components/velero/restic/restic_test.go

```diff
@@ -0,0 +1,60 @@
+// Copyright 2021 The Lokomotive Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package restic_test
```

Comment:
This test can be changed to verify, that we validate tolerations if we make validation rule a method of Toleration type.

Reply:
I think making it a method allows everyone else using tolerations to benefit from it.

Reply:
its a method that is called when RenderTolerations is called.

Reply:
If render already returns error, it's good to have it there, we keep API simple, thanks!